### DISCUSSION TITLE
Unable to ignore RDS: Instance does not have IAM Authentication enable

### DISCUSSION THREAD

**Discussion by gespinal** [aquasecurity/tfsec#2160] [Help and suppports]
I'm unable to ignore the RDS IAM authentication missing.

Tried `#tfsec:ignore:AWS0176` but that does not help.

```
Result #1 MEDIUM Instance does not have IAM Authentication enabled 
...
  Rego Package builtin.aws.rds.aws0176
     Rego Rule deny
...
```

Do not get any rule and don't see any defined here: https://github.com/aquasecurity/tfsec/blob/master/rules.md

**Comment by boenshao** [comment]
I encountered the same problem today, it turns out the rule is defined in Rego and the ID is actually `AVD-AWS-0176`.
See [here](https://github.com/aquasecurity/trivy-checks/blob/a6ecb69049ebb0d3b2e9083d1b3a25a2ca944d02/checks/cloud/aws/rds/enable_iam_auth.rego#L10) for more details.
